src/styles/stars.scss

```diff
@@ -0,0 +1,14 @@
+.stars {
+  display: flex;
+  &__star {
+    width: 16px;
```

Comment:
Consider using SCSS variables for dimensions like `16px` to improve maintainability and consistency across your stylesheets.